### ISSUE TITLE
Do you add the data processed by data augmentation into the dataset?

### ISSUE THREAD

**Issue by lsh-med** [nikhilroxtomar/FANet#3]
I read your paper,The Table I show the number of data after data augmentation,but l don't see in the code.

**Comment by tyjcbzd** [comment]
Hello, I read your paper and code. I found that in your ResidualBlock. The SElayer in the structure diagram is behind the 3 x 3 convolution, but the SElayer in the actual code is behind the 1 x 1 convolution. Maybe this is a mistake?
![66aa4f6acf094b210447889eec9db74](https://github.com/nikhilroxtomar/FANet/assets/117035550/09abd96f-44d2-4d74-bf7f-8cca8c7a3ce6)

**Comment by ruti0823** [comment]
Hello, I have also noticed your issue and improved the code according to the network structure diagram, but found that the performance is not as good as the original code. Have you solved this problem?

**Comment by tyjcbzd** [comment]
> Hello, I have also noticed your issue and improved the code according to the network structure diagram, but found that the performance is not as good as the original code. Have you solved this problem?

No, maybe they use some tricks. The results can vary from devices to devices.

**Comment by moonmisso** [comment]
> Hello, I have also noticed your issue and improved the code according to the network structure diagram, but found that the performance is not as good as the original code. Have you solved this problem?

Hello, may I ask what improvements you have made, and what is the final performance? Mine is ten points off
